vimmasite/vimma/celeryconfig.py

```diff
@@ -5,6 +5,10 @@
 _every_5min = crontab(minute='*/5')
 _every_1h = crontab(minute=0)
 
+BROKER_BACKEND = "redis"
```

Comment:
We'd be replacing one server with another, and tweaking the working project setup. I'd keep rabbitmq, it's worked well so far (dev & prod).